src/core/templates/show-people.js

```diff
@@ -62,113 +81,217 @@ const orcidIcon = () => html`<svg
-        class="ed_mailto u-email email p-name"
-        href="${`mailto:${p.mailto}`}"
+  const personName = [person.name]; // treated as opt-in HTML by hyperHTML
+  const company = [person.company];
+  const editorId = person.w3cid ? parseInt(person.w3cid, 10) : null;
+  const contents = [];
+  if (person.mailto) {
+    contents.push(html`<a
+      class="ed_mailto u-email email p-name"
+      href="${`mailto:${person.mailto}`}"
+      >${personName}</a
+    >`);
+  } else if (person.url) {
+    contents.push(
+      html`<a class="u-url url p-name fn" href="${person.url}"
         >${personName}</a
-      >`);
-    } else if (p.url) {
-      contents.push(
-        html`<a class="u-url url p-name fn" href="${p.url}">${personName}</a>`
-      );
-    } else {
-      contents.push(html`<span class="p-name fn">${personName}</span>`);
-    }
-    if (p.orcid) {
-      contents.push(
-        html`<a class="p-name orcid" href="${p.orcid}">${orcidIcon()}</a>`
-      );
-    }
-    if (p.company) {
-      if (p.companyURL) {
-        contents.push(
-          html`
-            (<a class="p-org org h-org h-card" href="${p.companyURL}"
-              >${company}</a
-            >)
-          `
+      >`
+    );
+  } else {
+    contents.push(html`<span class="p-name fn">${personName}</span>`);
+  }
+  if (person.orcid) {
+    contents.push(
+      html`<a class="p-name orcid" href="${person.orcid}">${orcidIcon()}</a>`
+    );
+  }
+  if (person.company) {
+    const hCard = "p-org org h-org";
+    const companyElem = person.companyURL
+      ? html`<a class="${hCard}" href="${person.companyURL}">${company}</a>`
+      : html`<span class="${hCard}">${company}</span>`;
+    contents.push(html` (${companyElem})`);
+  }
+  if (person.note) contents.push(document.createTextNode(` (${person.note})`));
+  if (person.extras) {
+    person.extras
+      .map(extra => html`, ${renderExtra(extra)}`)
+      .reduce((contents, html) => {
+        contents.push(html);
+        return contents;
+      }, contents);
+  }
+  if (person.retiredDate) {
+    const { retiredDate } = person;
+    const time = html`<time datetime="${retiredDate}"
+      >${humanDate(retiredDate)}</time
+    >`;
+    contents.push(html` - ${l10n.until(time)} `);
+  }
+  const dd = html`<dd
+    class="editor p-author h-card vcard"
+    data-editor-id="${editorId}"
+  >
+    ${contents}
+  </dd>`;
+  return dd;
+}
+
+function renderExtra(extra) {
+  const classVal = extra.class || null;
+  const { name, href } = extra;
+  return href
+    ? html`<a href="${href}" class="${classVal}">${name}</a>`
+    : html`<span class="${classVal}">${name}</span>`;
+}
+
+/**
+ * @param {Person} person
+ * @param {Number} index
+ * @returns
+ */
+function validatePerson(person, index) {
+  const hint =
+    "See [Person](https://respec.org/docs/#person) configuration for available options.";
+  if (!person.name) {
+    const msg = `Person object at index ${index} is missing required member "name"`;
```

Comment:
I'm not sure if `person object at index ${index}` is useful as an error message. The Person could be coming from either `respecConfig.editors`, `respecConfig.authors` or `respecConfig.formerEditors`.
We should either add the respecConfig key to the error message, or maybe render `JSON.stringify(person)` there.

Reply:
Yeah, was thinking the same... I'll add the name of the property as an argument.